Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneEventCollector/README.md

```diff
@@ -0,0 +1,92 @@
+Palo Alto Networks Trend Micro Vision One Event Collector integration for Cortex XSIAM collects the Workbench, Observed Attack Techniques, Search Detections and Audit logs.
+Trend Micro Vision One is a purpose-built threat defense platform that provides added value and new benefits beyond XDR solutions, allowing you to see more and respond faster. Providing deep and broad extended detection and response (XDR) capabilities that collect and automatically correlate data across multiple security layers—email, endpoints, servers, cloud workloads, and networks—Trend Micro Vision One prevents the majority of attacks with automated protection.
+## Configure Trend Micro Vision One Event Collector on Cortex XSIAM
+
+1. Navigate to **Settings** > **Configurations** > **Data Collection** > **Automation & Feed Integrations**.
+2. Search for Trend Micro Vision One Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| Your server URL | The api endpoint to the trend micro vision one instance, see domains list: https://automation.trendmicro.com/xdr/Guides/First-Steps-Toward-Using-the-APIs | True |
+| Trend Micro Vision One API Key | The Trend Micro Vision One API Key. Refer to the help section or to the information below on how to retrieve the API key. | False |
+| The maximum number of events per fetch | The maximum number of events to fetch every time fetch is executed for a single log-type \(Workbench, Observed Attack Techniques, Search Detections and Audit logs\). | False |
+| Observed attack techniques and Search detections logs date-time range (hours) | Defines the date-range (hours) in each api call that retrieves observed attack techniques and search detections logs. Used mainly to prevent timeouts by looping over large amount of logs.  Default is 24 hours. | False |
+| First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) |  | False |
```

Comment:
lets add descriptions to these

Reply:
its auto generated by `demisto-sdk generate-docs`, what should i add here?

Reply:
see what other integrations use, theyre pretty standard